Scalar.FunctionalTests/Tools/ScalarFunctionalTestEnlistment.cs

```diff
@@ -193,6 +193,11 @@ public string RunVerb(string task, long? batchSize = null, bool failOnError = tr
             return this.scalarProcess.RunVerb(task, batchSize, failOnError, asService);
         }
 
+        public string RunMaintenanceTask(string task, string config = null)
+        {
+            return GitProcess.Invoke(this.RepoRoot, $"{config}maintenance run --task={task}");
```

Comment:
Is it maybe too easy to forget the trailing space in the `config` parameter? I would think that adding a space after `{config}` would make things more robust in the long run.

Reply:
Possibly, but also this is test code so that implementer would notice that their test fails to run the Git command.